examples/stable/foundationdb/README.md

```diff
@@ -0,0 +1,248 @@
+# FoundationDB
+
+According to the foundationDB documentation FoundationDB gives us the power of ACID
+transactions in a distributed database and has below properties
+
+* Multi-model data store
+* Easily scalable and fault tolerant
+* Industry-leading performance
+
+**Note**
+These steps are taken from the steps that are mentioned in the official
+repository of [foundationDB operator](https://github.com/foundationdb/fdb-kubernetes-operator).
+Since they don't have any any way to install the operator without
+building the operator image, we have followed the the upstream `Makefile`
+to create this README that can be used to create a simple foundationDB
+cluster.
+
+
+## Prerequisite 
+
+* Install GO on your machine, see the [Getting Started](https://golang.org/doc/install) guide for more information.
+* Install KubeBuilder and its dependencies on your machine, see [The KubeBuilder Book](https://book.kubebuilder.io/quick-start.html) for more information.
+* Set your $GOPATH.
```

Comment:
@viveksinghggits Does this step really required?

Reply:
Hi @PrasadG193 
Yes `kubebuilder` should be installed on the machine.